collections-csharp/ConcurrentQueueInCSharp/ConcurrentQueueInCSharp.Console/OrderMessageBus.cs

```diff
@@ -0,0 +1,30 @@
+﻿using ConcurrentQueueInCSharp.Models;
+using System.Collections.Concurrent;
+
+namespace ConcurrentQueueInCSharp
+{
+    public class OrderMessageBus
+    {
+        private readonly ConcurrentQueue<Order> _queue = new ConcurrentQueue<Order>();
+
+        public int Count
```

Comment:
We can use an expression bodied property here

Reply:
Good point.